contracts/marketplace/src/query.rs

```diff
@@ -221,6 +225,37 @@ pub fn query_ask_renew_price(
     Ok((Some(coin(renewal_price.u128(), NATIVE_DENOM)), valid_bid))
 }
 
+pub fn query_ask_renew_prices(
+    deps: Deps,
+    current_time: Timestamp,
+    token_ids: Vec<String>,
+) -> StdResult<Vec<(Coin, Bid)>> {
+    token_ids
+        .iter()
+        .map(|token_id| {
+            let (coin_option, bid_option) =
+                query_ask_renew_price(deps, current_time, token_id.to_string()).map_err(|e| {
+                    StdError::generic_err(format!(
```

Comment:
The rest of the contract is using typed errors instead of generic errors so would be nice to be consistent.

Reply:
@shanev the function this calls (`query_ask_renew_price()`) uses a generic error and in the process of looking over this PR comment I realized that I was just repeating the same generic error response...what about this instead?

```rust
pub fn query_ask_renew_prices(
    deps: Deps,
    current_time: Timestamp,
    token_ids: Vec<String>,
) -> StdResult<Vec<(Coin, Bid)>> {
    token_ids
        .iter()
        .map(|token_id| {
            let (coin_option, bid_option) =
                query_ask_renew_price(deps, current_time, token_id.to_string())?;
            let coin = coin_option.unwrap();
            let bid = bid_option.unwrap();
            Ok((coin, bid))
        })
        .collect::<StdResult<Vec<_>>>()
}
```
...and just naturally unwrap the error as it happened (if im using unwrap correctly here...still tangled up in rust terminology)?

Reply:
...yeah so I just yanked that out and am yolo bubbling the error up through the `.map()` in the last set of changes